src/components/tasks/SuggestionBox/SuggestionList.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import classNames from '@/components/tasks/SuggestionBox/suggestion.module.scss';
+
+type Props = {
+    suggestions: GithubInfo[];
+    onClickName: (userName: string) => void;
+};
+
+const SuggestionList: FC<Props> = ({ suggestions, onClickName }) => {
```

Comment:
onClickName function is expecting a string as its argument, but the innerText property of an HTML element returns a DOMText node. This means that the onClickName function will receive a DOMText node instead of a string, which will cause an error.